core/templates/third-party-imports/midi-js.import.ts

```diff
@@ -16,12 +16,12 @@
  * @fileoverview This file imports the MIDI.js library.
  */
 
-require('static/midi-js-c26ebb/build/MIDI.js');
-window.Base64Binary = require('static/midi-js-c26ebb/inc/shim/Base64binary.js');
+import 'midi/build/MIDI.js';
+window.Base64Binary = require('/node_modules/midi/inc/shim/Base64binary.js');
 
 $(document).ready(function() {
   MIDI.loadPlugin({
-    soundfontUrl: '/third_party/static/midi-js-c26ebb/examples/soundfont/',
+    soundfontUrl: '/assets/audio/soundfont/',
```

Comment:
Changing it to `/node_modules/midi-js/examples/soundfont/` won't work? (This will help avoid keeping all the static files in our repo.)